src/components/PrivacyFilter/PrivacyFilter.tsx

```diff
@@ -41,6 +46,8 @@ export const PrivacyFilter: React.FunctionComponent = () => {
       reader.onabort = () => console.log("File reading was aborted.");
       reader.onerror = () => console.log("File reading has failed.");
       reader.onloadend = () => {
+        // Reset redaction list
+        setRedactionList([]);
```

Comment:
i was thinking of placing the sensitive fields checker right after this, so that it doesn't cause an infinite loop - is my understanding correct?